web/admin/components/Sbc/Modal/AnnualReportDetails/index.vue

```diff
@@ -114,13 +117,51 @@ const errorText = computed(() => useErrorMessage(error.value as ApiError, t('pag
                   $t('page.admin.table.invite.detailModal.section.reportData.noVote', { value: item.annualReport.votedForNoAGM ?? 'null' }),
                 ]"
               />
-              <UButton
-                class="self-start"
-                size="sm"
-                :label="showRawYears.includes(item.header.filingYear) ? $t('btn.viewRaw.close') : $t('btn.viewRaw.open')"
-                icon="i-mdi-code-json"
-                @click="handleViewRaw(item.header.filingYear)"
-              />
+              <UButtonGroup orientation="vertical">
```

Comment:
add vertical button group with tooltip for view json, download receipt, download report